src/Vektonn.DataSource/Kafka/KafkaConsumer.cs

```diff
@@ -81,20 +81,20 @@ public async Task RunAsync(CancellationToken cancellationToken)
                     {
                         if (error.IsFatal)
                         {
-                            LogExtensions.Error(this.log, $"ConfluentConsumer error: {error.ToPrettyJson()}");
+                            this.log.Error($"ConfluentConsumer error: {error.ToPrettyJson()}");
```

Comment:
Don't use `this` qualifier here, just `log`.

Reply:
https://github.com/vektonn/vektonn/pull/15/commits/9d27d19a072fa605366ddb4599bd207e8a319409